core/sail/extensible-store/pom.xml

```diff
@@ -22,16 +22,23 @@
             <artifactId>rdf4j-sail-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <version>4.4</version>
```

Comment:
We use older versions of commons-collections in several places - there's both 3.2.2 and 4.2 in use as (transitive) dependencies. Is it possible to reconcile this? Do you need 4.4 here specifically, or would 4.2 be good enough?

Reply:
Thanks Jeen. I’ll take a look and see if I can’t upgrade everything. There wasn’t much of a release note. They just simple wrote something along the lines of “bug fixes”.